pkg/app/app.go

```diff
@@ -116,6 +116,7 @@ func (c *App) Handler() http.Handler {
 	router.Handle("/oauth2callback", HTTPHandler(c.OAuth2Callback))
 	router.Handle("/deauth", c.Authenticate(c.DeAuthHandler)).Methods("GET")
 	router.Handle("/deauth", c.Authenticate(c.RescindAuthorizationHandler)).Methods("POST")
+	router.Handle("/info", c.Authenticate(c.InfoHandler)).Methods("GET")
```

Comment:
let's use the `/v1` prefix here as this logic  belongs to the REST API.

Reply:
Done